themes/default/layouts/migrate/terraform.html

```diff
@@ -136,6 +141,24 @@ <h5>Coexist with Terraform state</h5>
             </li>
         </ul>
     </section>
-
-    {{ partial "hand-raise-section" . }}
+    <section id="migrate" class="container mx-auto w-full relative overflow-visible py-20 px-4">
+        <div class="flex justify-center mt-24">
+            <div class="card ready-cta w-full rounded-xl p-1 gradient">
+                <div class="flex justify-center rounded-xl bg-purple-100">
```

Comment:
Ah sorry, missed this one!

```suggestion
                <div class="flex justify-center rounded-xl bg-violet-100">
```